examples/sol-anchor-contract/programs/example-sol-anchor-contract/src/lib.rs

```diff
@@ -0,0 +1,147 @@
+use std::mem::size_of;
+use anchor_lang::prelude::*;
+use solana_program::account_info::AccountInfo;
+use pyth_sdk_solana::load_price_feed_from_account_info;
+
+declare_id!("4ACQwuQt2ChnPRUV8KwuhVqJvEdusWd3i8JAngHDkp91");
+
+#[account]
+pub struct AdminConfig {
+    pub is_initialized:           bool,
```

Comment:
Anchor already knows if the account is initialized or not based on the first 8 bytes so you don't need to double check with this `is_initialized` field.